templates/plugin/host_reports.yml.erb

```
@@ -0,0 +1,23 @@
+---
+# Use https for production deployments. http and true only make sense in development
+:enabled: <%= @module_enabled %>
+
+# Proxy hostname to appear in reports JSON
+:reported_proxy_hostname: <%= scope.lookupvar('::foreman_proxy::plugin::host_reports::reported_proxy_hostname') %>
```

It's no longer needed to use `::foreman_proxy`. That was a thing for Puppet 3 and Puppet 4 to behave in the same way but thankfully we've dropped Puppet 3 a long time ago.
```suggestion
:reported_proxy_hostname: <%= scope.lookupvar('foreman_proxy::plugin::host_reports::reported_proxy_hostname') %>
```